### ISSUE TITLE
Contrast issues (Accessibility)

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/calendar#7929]
### Steps to reproduce

Issues:
- Past events are translucent making them not have enough contrast:

<img width="411" height="46" alt="Image" src="https://github.com/user-attachments/assets/908cc9e4-6268-4eff-ad03-64149b9e2e0a" />

- Tentative icon is yellow on white and doesn't have enough contrast:

<img width="157" height="53" alt="Image" src="https://github.com/user-attachments/assets/6f13110b-16da-4367-8cb0-3806e48a1397" />

There may be other instances of bad contrast

### Expected behavior

Contrast respecting WCAG guidelines

### Actual behavior

Not enough contrast in some parts of the app

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

### List of activated apps

```Text

```

### Nextcloud configuration

```Text

```

### Web server error log

```Text

```

### Log file

```Text

```

### Browser log

```Text

```
